### PR TITLE
add keep in touch section

### LINKED ISSUE #93: Implement keep in touch section
image

Note from Jack: "For the BE part of this task, can you bring this up during dev sync so BEs know to implement this if not implemented yet? I plan on having a batch change of connecting to BE for anything that needs a BE connection in the landing page after we finish up the UI, so for the current issue, simply ensure BE gets the endpoint ready for us and don’t worry about the actual calling/posting of data to BE"

### PR DESCRIPTION
**Pull request by JamesCahyadi**
Created the FE of the keep in touch section. I just realized that Jack mentioned to not worry about the posting of the data to BE, so let me know if I should remove the handlePost function. It should properly send the name and email as JSON to the backend though (just need to change it to the correct endpoint). Close #93.

### CODE REVIEW

File: src/components/landing_page/KeepInTouch.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
```
Comment (kerry918) [inline src/components/landing_page/KeepInTouch.js:1]: Maybe we could move the KeepInTouch section into its own folder (the landing page folder is getting large)

**Review by kerry918 — APPROVED** [review]
Leave a suggestion, otherwise looks great!

**Review by hyunzeekim — APPROVED** [review]
LGTM

File: src/components/landing_page/KeepInTouch.js (hunk 1 above)
Comment (jack-lin-jql) [reply src/components/landing_page/KeepInTouch.js:1]: +1

File: src/components/landing_page/KeepInTouch.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    { placeholder: "Full Name*", type: "text", name: "name" },
+    { placeholder: "Email*", type: "email", name: "email" },
+  ];
+
+  const handlePost = (e) => {
+    e.preventDefault();
+    const body = { name, email };
+
+    // TODO: change this url to match BE
+    fetch("http://localhost:8080/", {
```
Comment (jack-lin-jql) [inline src/components/landing_page/KeepInTouch.js:21]: Can you please create an issue to track this?

Comment (jack-lin-jql) [reply src/components/landing_page/KeepInTouch.js:21]: Additionally, no action item needed here, but for your information, JS fetch is special in a way that it doesn't reject the same way as one would expect upon the Promise return. See here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#checking_that_the_fetch_was_successful 

Additionally, we should add better error handling to this in the future, perhaps as a part of the issue mentioned above

**Review by jack-lin-jql — APPROVED** [review]
LGTM please address the comments accordingly

**Pull request merged**